ngsw-config.json

```diff
@@ -42,9 +42,9 @@
       }
     },
     {
-      "name": "childPhotos",
+      "name": "attachments",
       "urls": [
-        "**/child-photos/**"
+        "**/app-attachments/**/**"
       ],
       "cacheConfig": {
         "maxSize": 500,
```

Comment:
Do these values (size 500 max age 10 days) still make sense? This means users will have an old image for up to 10 days.

Reply:
if I understand the docs correctly, after remaining 10 days offline the image will not be displayed at all anymore ("considered invalid and eviced": https://angular.io/guide/service-worker-config#maxage)? In that sense, a big maxAge makes sense for offline support.

Maybe we should change this to the mentioned "staleWhileRevalidate" strategy though? In combination with our in-memory cache that could be a better fit?